MobileTraining/MobileTraining/Views/Login/LoginView.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  LoginView.swift
+//  MobileTraining
+//
+//  Created by Svetoslav Donkov on 14.05.23.
+//
+
+import SwiftUI
+
+struct LoginView: View {
+    @Binding var isLoggedIn: Bool
+    @ObservedObject var loginViewModel: LoginViewModel
+    @State private var showingAlert = false
+    @State private var isSecure = true
+    
+    private let networkManager: NetworkManager
+    
+    init(isLoggedIn: Binding<Bool>, loginRepository: LoginRepository,
+         networkManager: NetworkManager) {
+        self._isLoggedIn = isLoggedIn
+        self.loginViewModel = LoginViewModel(loginRepository: loginRepository)
+        self.networkManager = networkManager
+        
+        // Revoke access token in keychain
```

Comment:
You will need revocation only after implementing a `log out` in one of the last homeworks.